### DISCUSSION TITLE
Share / Sync transactions between two asset accounts?

### DISCUSSION THREAD

**Discussion by shyonae** [firefly-iii/firefly-iii#9028] [Firefly III]
HI everyone, I was just wondering what I wrote in the title.

I recently got a credit card, but it's connected to my bank account so the transactions are linked, basically. I still would like to create a different asset account on Firefly to keep myself on a budget, how may I go and sort of sync the transactions so that if I spend money with my credit card, it automatically adds the transaction to my bank account asset account too?

Am I over-complicating things? How would you do it? Thanks for your time!

**Comment by JC5** [comment]
Are you looking for something like the data importer? Of a script or something?

**Comment by shyonae** [comment]
Mh no I always put my transactions manually, the thing is that these two assets are linked, wdym a script? Can I do some sort of custom script?

**Reply by JC5** [reply]
Of course! Check out the documentation for webhooks, or perhaps the API. I think both can be of great help.
